### DISCUSSION TITLE
Duplicate Vertical Labels

### DISCUSSION THREAD

**Discussion by tonyofrancis** [patrykandpatrick/vico#436] [Questions]
How do I prevent duplicate vertical labels on the vertical axis? My data set sometimes can have the same value.

![Screenshot 2023-10-15 at 3 38 01 PM](https://github.com/patrykandpatrick/vico/assets/7353160/bf8ae2a0-d32e-4307-8eb4-6b11f26fd164)

**Comment by patrickmichalik** [comment]
Hello! It looks like your `AxisValueFormatter` returns the same label for different inputs. You can use the `maxItemCount` parameter of `AxisItemPlacer.Vertical.default` to limit the label count to such a value that the differences between the labels’ _y_ values are large enough for the `AxisValueFormatter` not to produce duplicate labels. Alternatively, for greater flexibility, you can create a custom `AxisItemPlacer.Vertical` implementation.
